### PR TITLE
Update data for design team

### PR DESCRIPTION
**Pull request by jasonyavorsky**

### CODE REVIEW

File: data/team.yml [hunk 1]
```diff
@@ -233,7 +233,8 @@ kai_passo:
 rob_rhyne:
   name: Rob Rhyne
   reports_to: director_design
-  role: Senior Product Designer, Batch Changes and Search
+  manager_role_slug: design_manager_code_graph
```
Comment (jasonyavorsky) [inline data/team.yml:236]:
```suggestion
  manager_role_slug: director_design
```

File: data/team.yml [hunk 2]
```diff
@@ -233,7 +233,8 @@ kai_passo:
 rob_rhyne:
   name: Rob Rhyne
   reports_to: director_design
-  role: Senior Product Designer, Batch Changes and Search
+  manager_role_slug: director_design
```
Comment (jasonyavorsky) [inline data/team.yml]:
```suggestion
  manager_role_slug: design_manager_code_graph
```

**Comment by jasonyavorsky** [comment]
@rrhyne please confirm this is correct before I merge (once we get confirmation from Christina).

File: data/team.yml [hunk 3]
```diff
@@ -2202,7 +2203,7 @@ amy_johnson:
   description: Amy lives in Southern California with her husband and mini australian shepherd. When she isn't knee deep in spreadsheets, she enjoys hiking, camping, and snowboarding.
 
 director_design:
-  name: Christina Forney
+  name: Christina Forney, Jason Yavorska (Code Graph only)
```
Comment (rrhyne) [inline data/team.yml]: I understand Christina to be the Director of Design as per: https://docs.google.com/document/d/1s5cqXzN_oG26ye3lCEEAYbjtAQ6hi4JGzu6VyQe44yQ/edit#

**Review by rrhyne — APPROVED** [review]

File: data/team.yml (hunk 3 above)
Comment (jasonyavorsky) [reply data/team.yml]: Ah, so you won't be reporting to me temporarily after all?

Comment (rrhyne) [reply data/team.yml]: I understand that you are my support for the transition to design team manager, but that I report to Christina.

Comment (jasonyavorsky) [reply data/team.yml]:
```suggestion
  name: Christina Forney
```

**Pull request merged**